rasa_core/policies/mapping_policy.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import os
+from typing import Any, List, Text, Optional
+
+from rasa_core.actions.action import (ACTION_LISTEN_NAME, ACTION_RESTART_NAME,
+                                      ACTION_BACK_NAME)
+
+from rasa_core import utils
+from rasa_core.domain import Domain
+from rasa_core.policies.policy import Policy
+from rasa_core.trackers import DialogueStateTracker
+
+logger = logging.getLogger(__name__)
+
+
+class MappingPolicy(Policy):
+    """Policy which maps intents directly to actions.
+
+    Intents can be assigned actions in the domain file which are to be
+    executed whenever the intent is detected. This policy takes precedence over
+    any other policy."""
+
+    def __init__(self, priority: Optional[int] = 5) -> None:
```

Comment:
```suggestion
    def __init__(self, priority: int = 5) -> None:
```
At least all of the other specified policies (not policy class) look like this